src/controllers/users.js

```diff
@@ -29,33 +30,44 @@ export async function registerUser(req, res) {
       email: user.email,
       hash: hash,
     });
-    const token = signToken(newUser)
-    newUser.setDataValue('token', token)
-    await newUser.save()
-    const responseData = userResponse(newUser)
+    const token = signToken(newUser.get());
+    newUser.setDataValue("token", token);
+    await newUser.save();
+    const responseData = userResponse(newUser.get());
     return res.status(201).json(responseData);
   } catch (e) {
     console.error(e);
-    if (e === "No payload found") return res.status(400).send("No payload found")
-    if (e === "Invalid payload format") return res.status(400).send("Invalid payload format, refer to docs for payload format")
-    if (e === "Invalid email format") return res.status(400).send("Invalid email format")
-    return res.status(500).send("Server error")
+    if (e.message === "No payload found")
+      return res.status(400).send(e.message);
+    if (e.message === "Invalid payload format")
```

Comment:
Lets combine these if statements for readability.

Reply:
I collected the errors into 1 if statement